proto/ai/traceable/agent/config/v1/config.proto

```diff
@@ -85,4 +88,9 @@ message ApiDiscoveryConfig {
 
 message SamplingConfig {
   google.protobuf.BoolValue enabled = 1;
+}
+
+message Javaagent {
+  // set this flag to export certificates configured in JKS to libtraceable for making HTTPS connection to TPA.
+  google.protobuf.BoolValue export_certs = 1;
```

Comment:
We can use `import_jks_certs` since this is only for JKS certs, and exporting may give a wrong idea that we are exporting certs somewhere else.